### ISSUE TITLE
Manage the password rules for my Nhost projects

### ISSUE THREAD

**Issue by gdangelo** [nhost/nhost#354]
Hello Nhost team,

It seems that the email/password authentication has some pre-defined rules. 

Indeed, in the screenshot below, which happens when calling the `useSignUpEmailPassword` or the `useChangePassword` React hooks, we can see that Nhost requires a password with at least eight characters by default.

So, I'm wondering if there is a way to see those rules somewhere in my Nhost dashboard and if I can configure them myself?

Also, I forgot to mention that the `error` returned by the hooks does not include an explicit message of why the authentication has failed. In other words, I had to look at the HTTP request from the console to know what was happening.

<img width="965" alt="Password rules" src="https://user-images.githubusercontent.com/4352286/162261428-f3384578-042f-4b13-88da-127c537c6e55.png">

Thanks,
Greg

**Comment by elitan** [comment]
That's right. We don't (yet) have support to change this value in Nhost Cloud.

**Comment by plmercereau** [comment]
Missing hasura-auth environment variables:
- AUTH_PASSWORD_HIBP_ENABLED
- AUTH_PASSWORD_MIN_LENGTH

**Comment by guicurcio** [comment]
I think that for this one (and several others) we should just remove the lock-up we have on adding certain variables (e.g. starting with AUTH\_, HASURA\_, etc...) we can add a subset that we allow the users to modify without having to wait for a UI change... wdyt @elitan @nunopato @elitan @szilarddoro

**Comment by szilarddoro** [comment]
The minimum password length is now configurable through the dashboard. I'm closing this issue now.
